src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RANDOM = 5;
+    private static final int RADIUS_RANDOM = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int inde = random.nextInt(FigureEnum.values().length);
+        FigureEnum figureEnum = FigureEnum.values()[inde];
+        Color color = colorSupplier.getRandomColor();
+        int sidr = random.nextInt(MAX_RANDOM) + 1;
+        switch (figureEnum) {
+            case CIRCLE:
+                return getRandomCircle();
+            case ISOSCELESTRAPEZOID:
+                return getRandomIsoscelesTrapezoid();
+            case RESTANGLE:
+                return getRandomRectangle();
+            case RIGHTTRIANGLE:
+                return getRandomRightTriangle();
+            case SQUARE:
+                return getRandomSquare();
+            default:
+                return null;
```

Comment:
[Checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-return-null-from-a-method)
Don’t return null from a method.
Returning null from a method is a bad practice. If you use switch-case in your solution, you may just put the last possible option in the default case.